net-im/toxic/toxic-0.8.3.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+
+inherit python-single-r1 xdg
+
+DESCRIPTION="A curses-based client for Tox"
+HOMEPAGE="https://github.com/JFreegman/toxic"
+SRC_URI="https://github.com/JFreegman/toxic/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="notification python qrcode +sound +video +X"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	dev-libs/libconfig
+	>=net-libs/tox-0.2.8
```

Comment:
That's not quite what my comment suggested ;)

Reply:
Did you mean that you would prefer the slot operator, or would you like the version to be 0.2.5.? If you meant that you preferred 0.2.5, the build fails with 0.2.5. It complains about undefined variable types. If you meant that you would prefer the slot operator, I will fix it and re-push.